gapir/cc/interpreter.h

```diff
@@ -181,10 +181,10 @@ class Interpreter {
   const MemoryManager* mMemoryManager;
 
   // The builtin functions.
-  std::unordered_map<uint8_t, FunctionTable> mBuiltins;
+  FunctionTable mBuiltins[16];
```

Comment:
Add a short comment about why it's `16` here.

Reply:
Done